specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -1079,13 +1079,52 @@
         }
       ],
       "properties": {
+        "identity": {
```

Comment:
There's definitely risk in adding this within an API version. If a user adds it in a recent client/portal, and a user with a legacy client does an update to the same resource (having no identity property as it was ), what will you do to the identity property? The proper behavior would be to remove `identity` value, but that is unlikely what is intended. By adding this in a new API version, you can have more intelligent handling of downlevel clients.